Bugsnag/Client/BugsnagClient.m

```diff
@@ -164,7 +164,9 @@ void BSSerializeDataCrashHandler(const BSG_KSCrashReportWriter *writer, int type
             writer->addJSONElement(writer, "state", bsg_g_bugsnag_data.stateJSON);
         }
         if (bsg_g_bugsnag_data.breadcrumbsPath) {
-            writer->addJSONFileElement(writer, "breadcrumbs", bsg_g_bugsnag_data.breadcrumbsPath);
+            // FIXME: This needs to be updated to cater for new breadcrumb storage scheme
+            // FIXME: addJSONFileElement generates broken JSON if the file cannot be opened
```

Comment:
Correction: addJSONFileElement generates broken JSON if passed the path to a directory